source/blog/2021-03-29-dbal-2.13.md

```diff
@@ -128,15 +128,12 @@ For a library, framework or platform:
 
 We recommend the following strategies to detect the use of deprecated code:
 
-1. If you have an extensive test-suite, register a PSR-3 logger with Doctrine
-   Deprecations to catch all deprecations while running the tests.
-2. Otherwise register a PSR-3 logger with Doctrine Deprecations in development
-   or staging only and collect and fix them as well you can. This could be
-   done for a longer amount of time of days, weeks or months.
-3. If that is not possible, register a PSR-3 logger in production. Make sure to
-   eliminate high frequency deprecations quickly or call `ignoreDeprecation` to
-   snooze them to avoid overhead.
-
-When you are part of the Symfony ecosystem and use the symfony/error-handler
-component, then you can also use the `trigger_error` mechanism instead of PSR-3
-logging.
+- Use Psalm or other static analyers to detect the use of Deprecated code.
```

Comment:
There's a typo in "analyers". "Deprecated" shouldn't be capitalized. We can mention IDEs here as well.